.storybook/config.js

```diff
@@ -38,7 +38,9 @@ addDecorator(story => (
     <br />
     <BpkRtlToggle />
     <br />
-    <BpkThemeToggle />
+    <div style={{ width: '10rem' }}>
```

Comment:
Unsure if there's a better way to do this, seeing as we don't have stylesheets for the storybook config (which also rules out using `TOKENS.onePixelRem * 10` because `string * 10 == NaN`

Reply:
`\`${parseInt(TOKENS.onePixelRem, 10) * 10}${TOKENS.onePixelRem.replace(/[0-9]/g, '')}\``

Reply:
🤣